test/shared_test/lib_utility_rate_test.cpp

```diff
@@ -183,8 +183,33 @@ TEST(lib_utility_rate_test, test_demand_charges_crossing_months)
 	int hour_of_year = 742; // 10 pm on Jan 31st
 	double cost = rate_forecast.forecastCost(forecast, 0, hour_of_year, 0);
 
-	// Total cost for the months would be $1511.25, but this subtracts off the peaks predicted by average load ($3.12)
-	ASSERT_NEAR(1508.11, cost, 0.02);
+	// Total cost for the months would be $1511.25, but this subtracts off the peaks predicted by average load ($1500)
+	ASSERT_NEAR(11.25, cost, 0.02);
```

Comment:
do you mean peaks predicted by gross load instead of "peaks predicted by average load"? Seems like with the forecast baseline using the gross peak instead of the net peak for demand, the difference of 11.25 here is due primarily to energy charges freshly incurred

Reply:
I think the most accurate statement would be "average gross load" - I'll update the comment. Yes, the $11.25 is due to energy charges.